apps/api/pages/api/bookings/_get.ts

```diff
@@ -125,7 +125,7 @@ function buildWhereClause(
   userEmails: string[] = []
 ) {
   const filterByAttendeeEmails = attendeeEmails.length > 0;
-  const userFilter = userIds.length > 0 ? { userId: { in: userIds } } : { userId };
+  const userFilter = userIds.length > 0 ? { userId: { in: userIds } } : !!userId ? { userId } : {};
```

Comment:
This formatting is horrendous. Will validate functionality and likely refactor.

Reply:
Would need to update the `userId` type in the function to `number | null`